envoy/api/v2/lds.proto

```diff
@@ -132,4 +132,8 @@ message Listener {
   // case the original source and destination addresses and ports are preserved on accepted
   // connections. Requires Envoy to run with the *CAP_NET_ADMIN* capability. Defaults to false.
   bool transparent = 10;
+
+  // Whether the listener should accept TCP Fast Open (TFO) connections.
+  // Defaults to false.
+  bool enable_tcp_fast_open = 11;
```

Comment:
@htuch @mattklein123 @ggreenway 

How do you feel about the naming convention for these listener options? Some are `use_` prefixed, some have no prefix, and this one is `enable_`. I chose `enable_` to indicate it is a bool, but I don't feel strongly about it at all.